libs/plugin.js

```diff
@@ -2,6 +2,7 @@ import axios from 'axios'
 import DrupalJsonApiEntity from './DrupalJsonApiEntity'
 import DrupalJsonApiTransformers from './DrupalJsonApiTransformers'
 import apiError from './DrupalJsonApiEntityError'
+import { cloneDeep } from 'lodash'
```

Comment:
I'd prefer to note use lodash's cloneDeep here since it will require the end user to have tree shaking in order to not include all of lodash. Lets swap this for `clone-deep` (https://www.npmjs.com/package/clone-deep) — to be fair, they should have tree shaking since this is nuxt....but still

Reply:
Cool, I'll swap it out and give it a test run to make sure it's behaving as expected